interface/main/calendar/add_edit_event.php

```diff
@@ -727,12 +738,20 @@ function setEventDate($start_date, $recurrence)
 
         // EVENTS TO FACILITIES
         $e2f = (int)$eid;
+        //Tell subscribers that a new multi appointment has been set
+        $patientAppointmentSetEvent = new AppoinmentSetEvent($_POST);
```

Comment:
AppoinmentSetEvent is mispelled: should be AppoinmentSetEvent here and in other placed below